source/configuration_manual/mail_location/obox/dictmap_cassandra_objectid.rst

```diff
@@ -155,8 +155,8 @@ Append the following to the ``dovecot-dict-cql.conf.ext`` file as described in
   map {
     pattern = shared/dictrevmap/$object_id/$object_name
     table = user_mailbox_objects_reverse
-    value_field = g
-    value_type = hexblob
+    value_field = u,g
+    value_type = string,hexblob
```

Comment:
Dangerous.. Doing this will break old versions. Also does the new version still work even without this config change? What if we changed "u" to be the second parameter, so even if someone did "g,u" with old version it wouldn't break?